### PR TITLE
Fixed Contacts component, add ContactsPage

### PR DESCRIPTION
**Pull request by Konfeta3-33**
[Ticket](https://trello.com/c/pvGpyNGs)
image

### CODE REVIEW

**Comment by Konfeta3-33** [comment]
@skyksandr, please have a look on my code

File: client/src/pages/ContactsPage.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Contacts from "../components/Contacts";
+import Footer from "../components/Footer";
+import HeaderBrown from "../components/HeaderBrown";
+
+const ContactsPage = () => {
+  return (
+    <>
+      <HeaderBrown />
+      <Contacts />
+      <Footer />
```
Comment (skyksandr) [inline client/src/pages/ContactsPage.js:11]: This looks repetitive and could be extracted to some kind of `MainLayout` component.

File: client/src/tailwind.css [hunk 2]
```diff
@@ -26,6 +26,14 @@
   height: 761px;
 }
 
+.contacts-block {
+  width: 760px;
```
Comment (skyksandr) [inline client/src/tailwind.css]: I wonder how it looks on screens more narrow than `760px + 760px + 3rem`

File: client/src/pages/ContactsPage.js (hunk 1 above)
Comment (Konfeta3-33) [reply client/src/pages/ContactsPage.js:11]: We will make it

File: client/src/tailwind.css (hunk 2 above)
Comment (Konfeta3-33) [reply client/src/tailwind.css]: fixed

**Pull request merged**